File: exercício1/exercício1-bsi.c
#include <stdio.h>

int main(void) {
  int x;
  printf("digite um número ");
  scanf("%d",&x);
  if ((x % 2) == 0){
    printf("PAR");
  }
  else{
    printf("IMPAR");
  }
  return 0;
}

File: exercício1/exercício1-bsi.pas
Program Pzim ;
	var
		x:integer;
	Begin
		writeln('Digite um número ');
		readln(x);
		if ((x mod 2)=0) then
			begin
				writeln('PAR');
			end
		else
			begin
				writeln('IMPAR');
			end;     
End.